### PR TITLE
anne week 4 notes

### PR DESCRIPTION
**Pull request by aleesteele**
Putting these here for the weekly meeting - I think it should cover everything from the past week!

### CODE REVIEW

**Comment by malvikasharan** [comment]
Thank you for all your thoughtful work Anne!

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 1]
```diff
@@ -6,6 +6,44 @@
...
+  - Turing Way coffee hours
+  - Turing Way reading group (read 1 section a week?)
+  - Slack automated welcoming message? (would love to co-create this, maybe in the community share out?)
+  - Github Issue templates  
+
+# Long term ideas
+- Review onboarding process 
+- Shared slack channels with other open science / related open source projects? (I've seen this happen on OSM)
+- UX/UI design sprint for section naming & organisation
+(just to keep my pie in the sky brain awake!)
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:42]:
```suggestion
- Review onboarding process - **MS**: Sounds fantastic. We should definitely create a process to be shared in the Community Handbook. So far it has happened through Coworking, Collab Cafe and workshops. As you know, I have used this doc in the past for onboarding calls: https://hackmd.io/@turingway/demo-intro (which is an overview of the GitHub and book): 
- Shared slack channels with other open science / related open source projects? (I've seen this happen on OSM) - **MS:** This is a Slack pro feature. We may not have funding for the pro account (as Turing institute has already a fully paid account and they will likely not approve extra workspace request) - but we can explore this with ABL if you can assess the pros and cons carefully.
- UX/UI design sprint for section naming & organisation
(just to keep my pie in the sky brain awake!) - **MS**: not sure what this means, happy to discuss! Wee probably need to have a UX/UI plan for the entire project (as well as merge the recent UX work by REG team.
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 2]
```diff
@@ -6,6 +6,44 @@
...
+
+# Where I need help
+- Maybe planning a timeline of things listed above, & prioritisation?
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:45]:
```suggestion
- Maybe planning a timeline of things listed above, & prioritisation? - **MS**: great! let's spend some time next week on this.
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 3]
```diff
@@ -6,6 +6,44 @@
...
 
+## 2022-04-26
+Trying a new format today to keep it more
+
+### Task updates
+- Getting reading for the fireside chat on friday! (thank you to all the speakers for being incredibly thoughtful - the event )
+- Community research forges on! (thank you to everyone I've spoken to, and will continue to speak to!)
+- Finally getting to work on some PRs this week (or at least understand themes in how people contribute for the community research update)
+- Community research will have more concrete tactics on slack engagement, onboarding, github
+
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:17]:
```suggestion

**MS:** Fantastic set of work. Thank you for carefully leading these.
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 4]
```diff
@@ -6,6 +6,44 @@
...
+### Upcoming event updates
+- frictionless data: 28 April (all set - really excited to see if there are ways of bringing in frictionless expertise to the project)
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:19]:
```suggestion
- frictionless data: 28 April (all set - really excited to see if there are ways of bringing in frictionless expertise to the project) - **MS:** All the best! This is exciting!
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 5]
```diff
@@ -6,6 +6,44 @@
...
+- fireside chat: 29 April (need to send some planning docs to LW - but should be all set to go)
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:20]:
```suggestion
- fireside chat: 29 April (need to send some planning docs to LW - but should be all set to go) - **MS:** Amazing! Looking forward to the discussions.
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 6]
```diff
@@ -6,6 +6,44 @@
...
+- newcastle: 9 may (postponed to September)
+- talk @ ssi: 19 may (sent recording, just waiting on planning)
+- book dash 16-20 may (reviewing applications this week.)
+- open gov week: 16-20 may (talking to someone from open heroines tomorrow - maybe good to get connected with them for this event?)
+- king’s college: 31 may (will send abstract by 29 april, reviewing MS templates, will chat with VK)
+- ordnance survey: 16 june (planning for early may follow up - waiting for follow-up)
+- connections turing course: late june-early july (following convo)
+
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:28]:
```suggestion

**MS:** thanks for the timeline and overview.
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 7]
```diff
@@ -6,6 +6,44 @@
...
+# Next week
+- More outreach to individuals
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:30]:
```suggestion
- More outreach to individuals. - MS: 💪🏽 
```

File: project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md [hunk 8]
```diff
@@ -6,6 +6,44 @@
...
+- Compile upcoming conferences to post on PR + start asking people if they'd like to rep TTW :) (would be super excited if possible!)
```
Comment (malvikasharan) [inline project-management/1-1-meetings/Anne-Lee-Steele/weekly-meetings.md:31]:
```suggestion
- Compile upcoming conferences to post on PR + start asking people if they'd like to rep TTW :) (would be super excited if possible!) - MS: we have generally aimed for at least 50% talks by community members. Keeping that or increasing that will be agood target. Do take some time tto look at the previous reports and promotion pack to think about what can be updated.
```

**Review by malvikasharan — APPROVED** [review]
Thank you so much Anne for the wonderful updates. All the best for a busy weeek, and some beautiful conversations to come.

**Comment by aleesteele** [comment]
Thank you, @malvikasharan ! I realise that I should have been merging these branches... woops. Now to merge the past few weeks :)

**Comment by aleesteele** [comment]
okay, actually i have to open it to merge. lol.

**Pull request merged**